packages/mdc-ripple/README.md

```diff
@@ -126,20 +126,19 @@ First import the ripple JS
 ##### ES2015
 
 ```javascript
-import {MDCRipple, MDCRippleFoundation} from 'mdc-ripple';
+import { MDCRipple, MDCRippleFoundation } from '@material/ripple';
```

Comment:
nit: This would actually fail our eslint configuration. Please conform to `{Component} from @material/component`